gtsfm/ui/gtsfm_process.py

```diff
@@ -0,0 +1,68 @@
+"""
+Defines GTSFMProcess, which holds a UiMetadata object for displaying this process in the process graph.
+
+Author: Kevin Fu
+"""
+
+# When creating a new class, follow this template:
```

Comment:
multi-line comment should go into triple-quote separated lines

Reply:
I wanted to do that at first, but to make the method as easy to copy-paste as possible I included the docstring of `get_ui_metadata()` in the comment block. (See line 19-20 [below](https://github.com/borglab/gtsfm/pull/540/files/c7ae439db14f58a05e7754a8707bac0ee9dd136d#diff-2100374802706ad206882380c8fed0951f0edd7a4ca5c88eb890105f20b53c7dR19-R20).) 

Do you have any workarounds for that?